tests/transactional/bootloader.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic smoke test to make sure that system can boot after
+#           few operations related to boot files and configurations
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base "opensusebasetest";
+use strict;
+use warnings;
+use testapi;
+use transactional;
+
+
+sub action {
+    my ($target, $text) = @_;
+    record_info('TEST', $text);
+    trup_call($target);
+    check_reboot_changes;
+}
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    action('bootloader',    'Reinstall bootloader');
+    action('setup-selinux', 'Enable SELinux') if (script_run("selinuxenabled") != 0);
+    assert_script_run('selinuxenabled');
+    action('grub.cfg',   'Regenerate grub.cfg');
+    action('initrd',     'Regenerate initrd');
+    action('kdump',      'Regenerate kdump');
```

Comment:
`kdump` doesn't have an impact on the bootloader, it just regenerates the kdump initrd